src/styles/platform-navigator.scss

```
@@ -0,0 +1,85 @@
+.platform-navigator {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  &__button {
+    display: flex;
+    justify-content: space-between;
+  }
+  &__dropdown {
+    border: 1px solid var(--amplify-colors-teal-80);
+    border-radius: 10px;
+    list-style-type: none;
+    padding: 0;
+    visibility: hidden;
+    z-index: 999;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    right: 0;
+    &--show {
+      visibility: visible;
+    }
+    &__item {
+      &:first-of-type{
+        border-top-left-radius: 10px;
+        border-top-right-radius: 10px;
+      }
+      &:last-of-type{
+        border-bottom-left-radius: 10px;
+        border-bottom-right-radius: 10px;
+      }
+      background-color: var(--amplify-colors-white);
+      &:hover {
+        background-color: var(--amplify-colors-teal-40);
+      }
+      & span {
+        padding: 0.5em;
+      }
+      &--current {
+        background-color: var(--amplify-colors-teal-20);
+      }
+    }
+  }
+}
+
+.info-popover {
+  .popover {
+    width: 22rem;
+    left: auto;
+    right: auto;
+    top: calc(100% + 1rem);
+    margin-left: -11rem;
```

Would it be more centered under the "Info" text if we did `-10.5rem`?
```suggestion
    margin-left: -10.5rem;
```